validators.go

```diff
@@ -160,3 +180,28 @@ var defaults = map[string]string{
 	strings.ToLower("VisualHostKey"):      "no",
 	strings.ToLower("XAuthLocation"):      "/usr/X11R6/bin/xauth",
 }
+
+// these identities are used for SSH protocol 2
+var defaultProtocol2Identities = []string{
+	"~/.ssh/id_dsa",
+	"~/.ssh/id_ecdsa",
+	"~/.ssh/id_ed25519",
+	"~/.ssh/id_rsa",
+}
+
+// these directives support multiple items that can be collected
+// across multiple files
+var multifileDirectives = map[string]bool{
+	"CertificateFile": true,
+	"IdentityFile":    true,
+	"DynamicForward":  true,
+	"RemoteForward":   true,
+	"SendEnv":         true,
+	"SetEnv":          true,
+}
+
+// IsMultifileDirective indicates a directive that can be found
```

Comment:
From OSX 10.11 for SendEnv:

> Variables are specified by name, which may contain wildcard characters.  Multiple environment variables may be separated by whitespace or spread across multiple SendEnv directives.  The default is not to send any environment variables.

I don't know where SetEnv came from, I can remove it.

Reply:
Actually, [here's where it came from](https://github.com/openssh/openssh-portable/blob/8a85f5458d1c802471ca899c97f89946f6666e61/readconf.c#L283). You can see from the [config parsing source code](https://github.com/openssh/openssh-portable/blob/8a85f5458d1c802471ca899c97f89946f6666e61/readconf.c#L1428) that it will keep adding additional directives as it encounters them.